instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -245,7 +246,8 @@ public Instrumenter<REQUEST, RESPONSE> build() {
                 statusExtractorTransformer.apply(HttpSpanStatusExtractor.create(attributesGetter)))
             .addAttributesExtractor(httpAttributesExtractorBuilder.build())
             .addAttributesExtractors(additionalExtractors)
-            .addOperationMetrics(HttpClientMetrics.get());
+            .addOperationMetrics(HttpClientMetrics.get())
+            .setSchemaUrl(SchemaUrls.V1_37_0);
```

Comment:
I think it makes sense to set schema url to the most recent version of schema that the instrumentation conforms to(?)

cc @lmolkova

Reply:
I've added this to next week's Semantic Convention SIG meeting to get feedback